### DISCUSSION TITLE
Problème télémétrie Crossfire

### DISCUSSION THREAD

**Discussion by swend24** [alexey-gamov/opentx-quad-telemetry#6] [Q&A] [answered]
Bonjour, je n'arrive pas à avoir la tension de la lipo, ni le timer, ni les modes de vol.

Comment dois-je procédé au niveau des réglages du script?
Je ne dois pas mis prendre correctement.

_J'ai une QX7 sous OpenTX et Crossfire pour Betafight et le tout avec les dernières mises a jours._

**Comment by swend24** [comment]
J'ai relevé une erreur dans le code concernant la télémétrie сrossfire.

Le capteur de tension est noté **RxBt** au lieu de **BtRx**, une fois changer cela fonctionne.
Pour le timer, j'ai finalement trouvé où l'activer dans le model.

Seul les modes de vol ne fonctionne pas pour le moment.
Il doit également s'agir d'un réglage dans le model mais je ne trouve pas lequel

**Comment by alexey-gamov** [comment, marked as answer]
Salut! Le script devrait fonctionner tel quel sur la configuration crossfire.
J'ai trouvé des comportements intéressants sur le firmware du module français (https://github.com/iNavFlight/inav/issues/4998)!

Pour le fonctionnement complet du script sans flasher vers la version **EN**, vous aurez besoin de:

1. Changer le capteur de batterie de **RxBt** à **BtRx** sur cette [ligne](https://github.com/alexey-gamov/opentx-quad-telemetry/blob/master/quad.lua#L324)
2. Changer le capteur de mode de vol de **FM** à **PV** sur cette [ligne](https://github.com/alexey-gamov/opentx-quad-telemetry/blob/master/quad.lua#L333)

J'espère que ces changements vous aideront!

**Reply by swend24** [reply]
Merci beaucoup. Tout fonctionne correctement
